packages/components/src/Field-Next/Field.test.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { ReactNode } from 'react';
+import { AriaTextFieldOptions, useTextField } from '@react-aria/textfield';
+import { render, screen, within } from '@testing-library/react';
+
+import { Field } from './Field';
+
+// Setup
+// ---------------
+interface MockedTestFieldProps extends AriaTextFieldOptions<'input'> {
+  disabled?: boolean;
```

Comment:
Whats your plan about this prop? Does the user have to give the disabled prop to the Field + to the Textinput e.g.? Or give the disabled prop only to Field and from there to his children?

Reply:
It's only to style it differently. Can be passed by whoever uses <Field>

Reply:
Yes but maybe its confusing API to give disabled to Field and to Textinput, or not?

Reply:
ah, sorry, I thought that comment was about `<HelpText>` this will be passed down to the `<input>`